tests-utils/src/main/scala/tech/beshu/ror/utils/containers/EsContainerWithRorSecurity.scala

```diff
@@ -64,7 +64,7 @@ object EsContainerWithRorSecurity extends StrictLogging {
                                            rorConfig: ReadonlyRestPlugin.Config) = {
     DockerImageCreator.create(
       Elasticsearch.create(esVersion, esConfig)
-        .install(new ReadonlyRestPlugin(esVersion, rorConfig))
+        .install(new ReadonlyRestPlugin(esVersion, rorConfig, performInstallation = true))
```

Comment:
It doesn't look good here. We "install" ReadonlyRestPlugin here but we can set `performInstallation = false`